pyMBE.py

```diff
@@ -1847,7 +1848,7 @@ def find_bond_key(self, particle_name1, particle_name2, use_default_bond=False):
         bond_keys = [particle_name1 +'-'+ particle_name2, particle_name2 +'-'+ particle_name1 ]
         bond_defined=False
         for bond_key in bond_keys:
-            if bond_key in self.df.values:
+            if bond_key in self.df["name"].values:
```

Comment:
@1234somesh this change looks like you found a small bug in another function. Did you find any unexpected behavior when using the previous implementation?